definitions/npm/express_v4.x.x/flow_v0.32.x-/test_response.js

```diff
@@ -14,7 +14,7 @@ app.use('/response_api', (req: express$Request, res: express$Response) => {
     'Content-Type': 'application/json',
   });
   res.end('{}');
-  console.log('request: ', res.req);
+  res.req;
```

Comment:
Wow.  This works?  neat.  I was going to suggest:

```js
const r: $Request = res.req
```

But yours is even simpler 😄